src/main/java/io/confluent/kafkarest/ConsumerManager.java

```diff
@@ -202,7 +201,6 @@ public String createConsumer(String group, ConsumerInstanceConfig instanceConfig
 
       synchronized (this) {
         consumers.put(cid, state);
-        consumersByExpiration.add(state);
         this.notifyAll();
```

Comment:
Do we need this with the changed expiration thread logic?

Reply:
@dhruvilshah3  could you help me review the https://github.com/confluentinc/kafka-rest/pull/500? thank you